plugins/civchat2-paper/src/main/java/vg/civcraft/mc/civchat2/listeners/CivChat2Listener.java

```diff
@@ -52,28 +54,36 @@ public void onPlayerDeath(PlayerDeathEvent playerDeathEvent) {
 
 	@EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
 	public void onPlayerQuit(PlayerQuitEvent playerQuitEvent) {
-		playerQuitEvent.setQuitMessage(null);
+		playerQuitEvent.quitMessage(null);
 		for (Player p : Bukkit.getOnlinePlayers()){
 			if (settings.getShowLeaves(p.getUniqueId()) && !db.isIgnoringPlayer(p.getUniqueId(), playerQuitEvent.getPlayer().getUniqueId())){
 				if (playerQuitEvent.getPlayer().hasPermission("civchat2.leavejoinimmune")) {
 					continue;
 				}
-				p.sendMessage(playerQuitEvent.getPlayer().getDisplayName() + ChatColor.YELLOW + " has left the game");
+				p.sendMessage(playerQuitEvent.getPlayer().displayName()
+						.append(Component.text(" has left the game")
+								.color(NamedTextColor.YELLOW)
```

Comment:
Small thing: it's better to do `Component.text(" has left the game", NamedTextColor.YELLOW)` instead as then you aren't re-creating the component with the colour, but instead are colouring the original component. Just a small allocation optimisation.